tests/AutoReview/MutatorTest.php

```diff
@@ -44,6 +44,8 @@
  *
  * @coversNothing
  *
+ * @group infra
```

Comment:
I introduced it to all test cases that are under `AutoReview` to exclude those tests to be run on AppVeyor. What do you think of `@group auto-review` instead and adding a test to ensure all the test cases under `AutoReview` have this group?